### PR TITLE
Update node version to 14.15.5 (latest TLS release)

### PR DESCRIPTION
**Pull request by sbc100**
See https://github.com/WebAssembly/binaryen/issues/3551

### CODE REVIEW

File: emsdk.py
Comment (kripken) [inline emsdk.py:1739]: looks like debug code?

File: scripts/update_node.py
Comment (kripken) [inline scripts/update_node.py:43]: doesn't this break the test?

**Review by kripken — CHANGES_REQUESTED** [review]
lgtm aside from comments

File: emsdk.py
Comment (sbc100) [reply emsdk.py]: Yup.. will remove before landing.

File: emsdk.py
Comment (kripken) [inline emsdk.py:1739]: looks like `errlog` prints to stderr - that still seems excessive here?

File: scripts/update_node.py
Comment (sbc100) [reply scripts/update_node.py:43]: This fixed a bug where we were including two copies of node in windows.  We were calling zip with the existing zip file present which was adding the new "bin" directory, but not removing the old zip contents.   So it saves a bit of space for all windows users.

File: emsdk.py
Comment (sbc100) [reply emsdk.py:1739]: Sorry, I'm just debugging the windows failure.

**Review by kripken — APPROVED** [review]
lgtm (with logging removed and assuming the windows fix does not change anything)

**Comment by sbc100** [comment]
This test starts to fail with the upgrade:  test_node_code_caching

@kripken  can you verify that this test fails with recent versions of node ?   Is this a known issue?

**Comment by kripken** [comment]
@sbc100 Oh, yes - this is a known issue, node changed/removed the API we depended on, https://github.com/nodejs/node/issues/18265#issuecomment-622971547 . There is a note on this in settings.js.

We can just disable the test, I guess. I hope node adds a new API at some point.

**Comment by sbc100** [comment]
Ok I will land this and roll a PR to disable the test.

**Pull request merged**
